packages/forms-web-app/src/controllers/selected-appeal/section.d.ts

```diff
@@ -0,0 +1,10 @@
+import { AppealCaseWithAppellant } from '@pins/common';
```

Comment:
Do we want to import from here or the typedef way from appeals-service-api?  Not sure if it makes a difference, this current way is probably neater so feel free to ignore

Reply:
Good point! It might not actually be working importing like this but I thought I might get away with it since it's a ts file. I'll try to make sure it's actually getting this definition into Sections

Reply:
this import was _not_ working but it is now